src/main/resources/db/migration/V163__add_comon_platform_flag.sql

```diff
@@ -0,0 +1,113 @@
+ALTER TABLE public.search_inperson
+ADD COLUMN common_platform BOOLEAN DEFAULT FALSE;
+
+
+UPDATE public.search_inperson
+SET common_platform = true
+
+where court_id in (select id from public.search_court where slug in (
+'ealing-magistrates-court',
```

Comment:
and just a double check on this one, was this list checked by Pauline :) ?

Reply:
yeah all the courts that are on this page https://www.gov.uk/guidance/hmcts-common-platform-participating-criminal-courts are participating